### PR TITLE
GSdx: Adjust Spartan crc hack

### PR DESCRIPTION
**Pull request by ivan89el**
Less effects are missing (post-processing is restored and interface in 60hz mode). Add crc id CrashBandicootWoC RU

### CODE REVIEW

**Comment by lightningterror** [comment]
I'll need to test this or maybe @MrCK1 can.

File: plugins/GSdx/GSHwHack.cpp [hunk 1]
```diff
@@ -246,29 +246,17 @@ bool GSC_Spartan(const GSFrameInfo& fi, int& skip)
...
-					// General, often problematic post processing
-					(GSUtil::HasSharedBits(fi.FBP, fi.FPSM, fi.TBP0, fi.TPSM)) )
-				{
-					skip = 1;
-				}
+				skip = 2;
 			}
 		}
 	}
-
```
Comment (lightningterror) [inline plugins/GSdx/GSHwHack.cpp]: You didn't need to remove this line.

File: plugins/GSdx/GSHwHack.cpp [hunk 2]
```diff
@@ -246,29 +246,18 @@ bool GSC_Spartan(const GSFrameInfo& fi, int& skip)
...
+
```
Comment (lightningterror) [inline plugins/GSdx/GSHwHack.cpp]: Tab space not needed.

**Comment by ivan89el** [comment]
Old hack (dx11/ogl) https://i105.fastpic.ru/big/2018/0719/c1/_ba5bafb65f1765a88fedcf76523816c1.png
new hack (dx11/ogl) https://i105.fastpic.ru/big/2018/0719/6e/_9d7f281b4443ca4c434b93af75216f6e.png
Software https://i104.fastpic.ru/big/2018/0719/a9/_1ae92d7645dd3929d7d1af85f833eca9.png

**Comment by ivan89el** [comment]
The old hack removed too many effects and interface. The new removes only part of the bloom and yellow glow (these effects are not emulated correctly in either dx11 or ogl)

**Comment by lightningterror** [comment]
For me it didn't remove the ui but the yellow glow remained, I guess the ntsc hack was inaccurate.

Anyway it works much better now.

The game utilizes channel shuffle and 8bit shader so one of them could be wrong, I'll leave this here just in case if someone wants to look at it in the future

```
T:Error	ID:5877	S:Mid	=> Channel shuffle effect detected
T:Error	ID:5877	S:Mid	=> Reading RT as a packed-indexed 8 bits format
T:Error	ID:5877	S:Mid	=> Channel shuffle effect detected (2nd shot)
T:Error	ID:5877	S:Mid	=> Red channel
T:Error	ID:5961	S:Mid	=> ERROR: Source and Target are the same!
T:Error	ID:5964	S:Mid	=> ERROR: Source and Target are the same!
T:Error	ID:5967	S:Mid	=> ERROR: Source and Target are the same!
```

**Pull request merged**